.prettierignore

```diff
@@ -4,3 +4,4 @@ yearn/
 typescript/
 system-tests/
 monitoring/
+cross-chain/
```

Comment:
Regarding the top-level directory, It's also worth adding the new package to:
- https://github.com/keep-network/tbtc-v2/blob/main/.pre-commit-config.yaml
- https://github.com/keep-network/tbtc-v2/blob/main/README.adoc

Reply:
Readme part: [a812bc2](https://github.com/keep-network/tbtc-v2/pull/554/commits/a812bc2ed7032f657eca670cef90ce632413a185)